tdrs-backend/tdpservice/data_files/admin/admin.py

```diff
@@ -113,14 +113,16 @@ def queryset(self, request, queryset):
     ]
 
     list_filter = [
+        'id',
```

Comment:
won't this show *every* available `id`? at some point i imagine the list would get quite large. is there a point to having an `id` numeric filter like this? would a range or other filter mechanism be more useful?

Reply:
Alex confirmed it is unnecessary given what #3106 will provide.